src/containers/FavoritesWarningDialog/FavoritesWarningDialog.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
If you pull the favorites stuff out into its own button, this can be part of the button instead of its own component. As is, it's a bit awkward because you have to remember to put it with the button in order for the warnings to show up.

Reply:
Done